reconcile/aus/base.py

```diff
@@ -207,15 +216,13 @@ def expose_org_upgrade_spec_metrics(
 class GateAgreement(BaseModel):
     id: str
 
-    def create(self, ocm: OCM, cluster_name: str) -> None:
-        logging.info(
-            f"create agreement for gate {self.id} on cluster {cluster_name} in OCM org {ocm.name}"
-        )
-        agreement = ocm.create_version_agreement(self.id, cluster_name)
+    def create(self, ocm_api: OCMBaseClient, cluster_id: str) -> None:
+        logging.info(f"create agreement for gate {self.id} on cluster {cluster_id}")
```

Comment:
I am not sure this log will be very easy to parse. Cluster names are easier than id for that matter

Reply:
i will add the cluster_name but the ID gets more important when we act in lots of orgs where the cluster name is not unique anymore

Reply:
[66733c0](https://github.com/app-sre/qontract-reconcile/pull/3655/commits/66733c087c4661b31d324c4b563602c7e3315a6d)

Reply:
maybe add the org id (or name once we can ensure we have a meaningful one..)